core/src/main/java/cucumber/runtime/formatter/PluginFactory.java

```diff
@@ -199,7 +199,7 @@ private Appendable defaultOutOrFailIfAlreadyUsed(String formatterString) {
 
     public static boolean isFormatterName(String name) {
         Class pluginClass = getPluginClass(name);
-        return Formatter.class.isAssignableFrom(pluginClass) || ConcurrentFormatter.class.isAssignableFrom(pluginClass);
+        return Formatter.class.isAssignableFrom(pluginClass);
```

Comment:
@mlvandijk is working on #1401 so I made ConcurrentFormatter extend the Formatter. This avoids changes to the PluginFactory.